src/guide/style.md

```diff
@@ -2,13 +2,93 @@
 
 > Capitalize the first word of every letter except articles, coordinating conjunctions, and prepositions of three letters or fewer. There’s one exception: Any word that is the first word in the headline or the last word should be capitalized, regardless of its part of speech.
 
-## Application and UI Labels
+Every page should begin with a level-1 heading, and heading levels should never be skipped. For example, do not use a level-3 heading below a level-1 heading without a level-2 heading in between.
+
+Use headings to segment content by topic. _Do not_ use headings for emphasis.
+
+Headings should also enable easy reference. When writing a heading, consider how well it describes the section's contents. If a reader saw the heading in a table of contents, would they have a general idea of what it contains?
+
+## Quotes
+
+Quotes have semantic meaning, and should therefore be used only in specific circumstances.
+
+The most obvious usage is to include a verbatim statement by another person or entity. In the case of a rhetorical or hypothetical statement, use italics.
+
+<aside class="aside-example">
+
+Ryan said, "I used to think less was more, but now I'm thinking more is more!"
+
+You might be thinking to yourself, *JavaScript sure is a crazy language!* And you'd be right.
+
+</aside>
+
+The only other proper use of quotes is to refer to a word as an object within a sentence.
+
+<aside class="aside-example">
+
+When it comes to words, "circumlocution" is one of the best.
+
+</aside>
+
+When using quotes in this way, all punctuation goes *outside* the quotes.
+
+Finally, when writing curriculum you should *avoid using scare quotes.* Scare quotes are quotation marks used around a word or phrase intended to signal irony or derrision.
+
+<aside class="aside-example">
+
+The Internet is built on HTML, a "programming language" introduced in 1993.
+
+</aside>
+
+The use of scare quotes is overtly colloquial and can be confusing. If tempted to use scare quotes, rethink what you want to convey and find a way to do it without them. Not convinced? [Read a few reasons why scare quotes should be avoided.](https://www.dailywritingtips.com/3-erroneous-uses-of-scare-quotes/)
+
+In all cases, use double quotes unless including a direct quotation that itself includes a quote.
+
+## Formatting for Emphasis
+
+To call attention to a sentence, phrase, or word within a paragraph, you may use italicized text.
+
+<aside class="aside-example">
+
+JavaScript *does not* allow you to specify the types of parameters, though the JavaScript extension TypeScript does.
+
+</aside>
+
+It is also acceptable to use all-caps to emphasize text. However, we prefer italics over all-caps in most situations, and all-caps should be avoided for phrases and sentences.
+
+<aside class="aside-example">
+
+JavaScript DOES NOT allow you to specify the types of parameters, though the JavaScript extension TypeScript does.
```

Comment:
When emphasizing a "not" idea, only that word should be in all caps.  So "DOES NOT" is better presented as "does NOT".